build_cnstrc_macos_wheel.sh

```diff
@@ -0,0 +1,4 @@
+./build_cnstrc_wheel.sh
```

Comment:
It will be good to add a shebang line in the beginning of this script so we know which interpreter it will use during execution. (I think bash is good, and that's what we do in most other places). 

Also worth adding `set -e` so the script fails if some of intermediate commands fails.
```suggestion
#!/bin/bash

set -e

./build_cnstrc_wheel.sh
```

Reply:
https://github.com/Constructor-io/kenlm/pull/2/commits/8bcd224781b260600f533b766811fef32c4fddf6